lib/ansible/modules/cloud/azure/azure_rm_storageblob.py

```diff
@@ -111,6 +111,11 @@
         choices:
             - container
             - blob
+    shared_access_key:
+        description:
+            - Use shared access key to authenticate your applications.
+            - Required when making requests to the storage account in other subscription.
```

Comment:
Required when I(resource_group) not defined

Reply:
done